stock_picking_move_package_to_package/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

Comment:
Please replace the writing with license in all files where it is indicated

now listed is http://www.gnu.org/licenses/lgpl
but it is necessary
http://www.gnu.org/licenses/agpl

Reply:
Thanks for noticing. I've fixed it. Please check.